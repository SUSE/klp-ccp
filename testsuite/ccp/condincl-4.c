void pu_f(void)
{
#if 0
#endif

#if 0
#endif

	0;

#if 0
#endif

#if 0
#endif

#if 1
#if 0

#endif
#if 0
#endif

	0;

#if 0
#endif

#if 0
#endif

#endif
}
