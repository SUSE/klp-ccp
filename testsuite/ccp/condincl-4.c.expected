/* klp-ccp: from cp/testsuite/ccp/condincl-4.c */
void klpp_pu_f(void)
{
#if 0
#error "klp-ccp: non-taken branch"
#endif

#if 0
#error "klp-ccp: non-taken branch"
#endif
	0;

#if 0
#error "klp-ccp: non-taken branch"
#endif

#if 0
#error "klp-ccp: non-taken branch"
#endif

#if 1
#if 0
#error "klp-ccp: non-taken branch"
#endif
#if 0
#error "klp-ccp: non-taken branch"
#endif
	0;

#if 0
#error "klp-ccp: non-taken branch"
#endif

#if 0
#error "klp-ccp: non-taken branch"
#endif

#else
#error "klp-ccp: a preceeding branch should have been taken"
#endif
}
